src/compressed_tensors/transform/factory/base.py

```diff
@@ -99,10 +99,10 @@ def _apply_to_module(self, module: Module, args: TransformArgs):
         # create transform as submodule
         transform_name = f"{self.name}_{args.location.value}"
         transform = self.create_transform(module, args)
-        register_offload_module(module, transform_name, transform)  # (1)
```

Comment:
If we're removing this from outside the if/else branches, wouldn't this mean for cases where the location is WEIGHT_INPUT/WEIGHT_OUTPUT, we're updating without first registering?